python/ray/train/examples/torch_linear_example.py

```diff
@@ -24,6 +24,11 @@ def __len__(self):
         return len(self.x)
 
 
+class LinearDatasetDict(LinearDataset):
```

Comment:
If this is only used in the test and not part of the example, can we move this to `test_gpu` directly?

Reply:
That makes sense, I've moved it just under the fixtures defined on `test_gpu`, just to keep the separation between the `test_*` functions and the rest. But if you want to have it defined just above `test_torch_prepare_dataloader`, I could do that too.